src/__Private/LintRunConfig.hack

```diff
@@ -82,6 +82,7 @@ final class LintRunConfig {
     HHAST\PocketEnumDeclarationLinter::class,
     HHAST\PreferRequireOnceLinter::class,
     HHAST\ConsistentLineEndingsLinter::class,
+    HHAST\PackagePrivateLinter::class,
```

Comment:
cc @jjergus @lexidor - IMO this one should go in NON_DEFAULT_LINTERS

Reply:
Curious: How is a non default linter activated?

Reply:
The `builtinLinters` options in the hhast config has three possible values. `all`, `default`, and `none`. When set to `all`, the non defaults are active. When set to `default`, the non defaults are excluded. `none` doesn't activate any linters. All modes allow `extraLinters` to be set as well.